docs/source/getting-started.mdx

```diff
@@ -219,9 +219,35 @@ Paste this string into the Operations panel and click the blue button in the upp
 
 <img class="screenshot" src="./images/sandbox-response.jpg" width="400" alt="Sandbox response panel"/>
 
+One of the most important concepts of GraphQL is that clients can choose to query _only for the fields they need_. Delete `author` from the query string and execute it again. The response updates to include only the `title` field for each book!
+
 > **Note:** If your server is deployed to an environment where `NODE_ENV` is set to `production`, introspection is **disabled** by default. This prevents Apollo Sandbox from working properly. To enable introspection, set `introspection: true` in [the options to `ApolloServer`'s constructor](./api/apollo-server/#constructor).
 
-One of the most important concepts of GraphQL is that clients can choose to query _only for the fields they need_. Delete `author` from the query string and execute it again. The response updates to include only the `title` field for each book!
+> **Note:** If you have a CORS configuration set up, your Apollo Server endpoint will need to whitelist
```

Comment:
Maybe just "if you have CORS configured"?

Reply:
whitelist -> safelist

Reply:
"You can choose instead" -> "Instead, you can choose"

Reply:
"on your endpoint" sounds a little funny, not sure what the right answer is. "for your endpoint"?

Reply:
TIL
https://www.grammarly.com/blog/comma-before-so/
>Don’t Use a Comma with “So” + a Dependent Clause

(remove comma between endpoint and so)